unitTesting/IO/gpio/Gpio.cpp

```diff
@@ -0,0 +1,44 @@
+#include <catch2/catch.hpp>
+
+#define USE_GPIO 1
+#include <IO/gpio/Gpio.h>
+
+#define DIGITAL_1 "Digital 1"
+#define DIGITAL_2 "Digital 2"
+#define PWM_1 "PWM 1"
+#define PWM_2 "PWM 2"
+
+class TestableGpio : public Gpio
+{
+    public:
+        TestableGpio() {};
+        ~TestableGpio() {};
+};
```

Comment:
Looks like we can't create `.h` files in the unit test folder right now, but this one is probably fine being inline.